### PR TITLE
Front init

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR:
- ensure all views are Vue.js based (allowing porting shared components)
- ensure all views have a common initialization
- fix and simplify common dependencies and style handling
- compile chunks in `udata/static/chunks` for easier cleanup

This is a start base for smaller PR.

### CODE REVIEW

File: js/front/apidoc.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/**
+ * Display a SwaggerUI documentation
+ */
+import 'front/bootstrap';
+
+import 'less/udata/swagger.less';
+
+import Vue from 'vue';
+
+import $ from 'expose?$!expose?jQuery!jquery';
```
Comment (davidbgk) [inline js/front/apidoc.js]: Just curious about that syntax.

File: js/front/apidoc.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import 'script!swaggerui/lib/underscore-min';
+import 'script!swaggerui/lib/backbone-min';
+import 'script!swaggerui/lib/jsoneditor.min';
+
+import 'script!swaggerui/swagger-ui.min';
+SwaggerUi = window.SwaggerUi;
+
+// Marked compatibility for SwaggerUI
+window.marked = commonmark;
+marked.setOptions = function() {};
```
Comment (davidbgk) [inline js/front/apidoc.js]: `_ => {}` do we turn the file into ES6 too?

File: js/front/user.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                currentTab: 0,
+                // API,
+            };
+        },
+        ready() {
+            log.debug('User display page');
+
+            // Restore tab from hash
+            if (location.hash !== '') {
+                this.$refs.tabs.$children.some((tab, index) => {
```
Comment (davidbgk) [inline js/front/user.js]: I see duplicated code here, can we factorize?

File: udata/templates/apidoc.html [hunk 4]
```diff
@@ -6,20 +6,13 @@
     'keywords': ['API', 'documentation', 'swagger'],
 } %}
 
+{% set bundle = 'apidoc' %}
```
Comment (davidbgk) [inline udata/templates/apidoc.html]: Does the page load/become responsive faster with that version?

**Comment by davidbgk** [comment]
We really need to sprint on adding tests for the frontend 😨

File: js/front/apidoc.js (hunk 1 above)
Comment (noirbizarre) [inline js/front/apidoc.js]: SwaggerUI requires jQuery to be a global. This syntax is from the webpack expose plugin (expose both `$` and `jQuery`)

File: js/front/apidoc.js (hunk 2 above)
Comment (noirbizarre) [inline js/front/apidoc.js]: It's already turned (there is import). The file has only been moved.

File: js/front/user.js (hunk 3 above)
Comment (noirbizarre) [inline js/front/user.js]: OK, but not in this PR

File: udata/templates/apidoc.html (hunk 4 above)
Comment (noirbizarre) [inline udata/templates/apidoc.html]: Not this page, it has been untouched

**Comment by noirbizarre** [comment]
Yes, there is some unit test (the specs folder) but we need end to end testing.
I think they should live outside the udata core repository as end to end tests relies on rendering and rendering is specific to each instance (theme, special pages...)

**Pull request merged**